### DISCUSSION TITLE
WPF Performance

### DISCUSSION THREAD

**Discussion by worthingtonjg** [Mapsui/Mapsui#2766] [Q&A]
I followed the getting started instructions for WPF, but I am noticing that the map is slow. Zooming is tolerable, but when I try to drag the map, to change the location, it is super slow - takes several seconds to respond to the drag.  Is this normal?  Is there anything I can do to make this work faster?

**Comment by pauldendulk** [comment]
This is not normal. You can get an impression of the performance in our Blazor samples, which is our slowest platform https://mapsui.com/v5/samples/
